### ISSUE TITLE
rider_type - linking with categories

### ISSUE THREAD

**Issue by antrim** [ODOT-PTS/GTFS-ride#24]
rider_type and rider_type description could be synced with [GTFS+ concepts](https://www.transitwiki.org/TransitWiki/images/e/e7/GTFS+_Additional_Files_Format_Ver_1.7.pdf) (or another such approach to fare categories, as generally, it seems likely that rider types should map onto fare categories). Alternately they could map to something else like census designated categories.

**Comment by carletop** [comment]
Yes, I believe such links would be helpful as a way to standardize the depiction of categories used across several realms, but perhaps further down the road. I wouldn't want a potential user of GTFS-ride to feel required to understand or use GTFS+ as a prerequisite for using the GTFS-ride standard. Is GTFS+ growing in adoption to where it would make sense to link to their designations? We definitely want to standardize as much as possible while balancing the need for flexibility which will enable the largest numbers of initial adopters of GTFS-ride.
